packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -1214,17 +1214,26 @@ class _InkResponseState extends State<_InkResponseStateWidget>
   }
 
   bool isWidgetEnabled(_InkResponseStateWidget widget) {
+    return _primaryButtonEnabled(widget) || _secondaryButtonEnabled(widget);
+  }
+
+  bool _primaryButtonEnabled(_InkResponseStateWidget widget) {
     return widget.onTap != null
       || widget.onDoubleTap != null
       || widget.onLongPress != null
       || widget.onTapUp != null
-      || widget.onTapDown != null
-      || widget.onSecondaryTap != null
+      || widget.onTapDown != null;
+  }
+
+  bool _secondaryButtonEnabled(_InkResponseStateWidget widget) {
+    return widget.onSecondaryTap != null
       || widget.onSecondaryTapUp != null
       || widget.onSecondaryTapDown != null;
   }
 
   bool get enabled => isWidgetEnabled(widget);
+  bool get primaryButtonEnabled => _primaryButtonEnabled(widget);
+  bool get secondaryButtonEnabled => _secondaryButtonEnabled(widget);
```

Comment:
What's the reason for making these public?  I'm not against it I just want to make sure we don't expose new APIs that could be private.

Reply:
I think there is no good reason and that these accessors should be private, good catch!
I have updated the PR accordingly.